src/gui/processing/models/qgsmodeldesignerdialog.cpp

```diff
@@ -835,6 +835,9 @@ void QgsModelDesignerDialog::populateZoomToMenu()
       QAction *zoomAction = new QAction( box.description(), mGroupMenu );
       connect( zoomAction, &QAction::triggered, this, [ = ]
       {
+        QRectF groupRect = item->boundingRect();
```

Comment:
Does this work? I'd have thought this needs to instead be:

```suggestion
        QRectF groupRect = item->mapToScene( item->boundingRect() );
```

Reply:
I'll try to rebuild the branch later today and add a screen cast but from my testing the other day, it worked.

Reply:
@nyalldawson Using your suggestion I get
```
../src/gui/processing/models/qgsmodeldesignerdialog.cpp: In lambda function:
../src/gui/processing/models/qgsmodeldesignerdialog.cpp:839:44: error: conversion from ‘QPolygonF’ to non-scalar type ‘QRectF’ requested
  839 |         QRectF groupRect = item->mapToScene( item->boundingRect() );
      |                            ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
ninja: build stopped: subcommand failed.
```
while with `QRectF groupRect = item->boundingRect();` or `QRectF groupRect = item->itemRect();` I get
![modelerZoomTo](https://user-images.githubusercontent.com/7983394/142051076-d834c22a-d6dc-47e8-99bf-ae26fab06021.gif)
t

Reply:
Argh, try this:

    QRectF groupRect = item->mapToScene( item->boundingRect() ).boundingRect();

Reply:
Yeah, it works. I guess I need to switch to your version ;) Willing to explain why it's better?

Reply:
It's really a future-proofing thing. Your original version works because there's no transforms currently applied to the model designer scene, but if in future we ever change that then we need the mapToScene to account for it.

Reply:
Euh... I did not commit your suggestion, yet. Or did you do it somehow?
Thanks for explaining.

Reply:
Ahhh oops, I assumed it was there. CAn you post a follow up PR?

Reply:
Done in #46125